.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.19.x]
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ matrix.go-version }}
+
+      - name: Checkout code
+        uses: actions/checkout@v3
+    
+      - name: Build
+        run: go build -v ./...
+
+      - name: Lint
+        uses: golangci/golangci-lint-action@v3
+        with:
+          version: latest
+          args: --no-config --disable-all --enable gosimple --enable govet --enable unused --enable deadcode --timeout 3m
+
+      - name: Run tests
+        run: go test -mod=vendor -v --race -covermode=atomic -coverprofile cover.out ./...
```

Comment:
Add an empty line.

Reply:
@SammyOina Please do not mark it as resolved but rather actually resolve comments.